assembl/models/mail.py

```diff
@@ -288,8 +288,8 @@ def strip_full_message_quoting_html(message_body):
         doc = None
         try:
             doc = html.fromstring(message_body)
-        except: # We can get "ParserError: Document is empty", which has to be caught, otherwise it blocks the SourceReader
-            return message_body
+        except etree.ParserError: # If the parsed HTML document is empty, we get a "ParserError: Document is empty" exception. So the stripped message we return is an empty string (if we keep the exception it blocks the SourceReader)
+            return ""
```

Comment:
That will silently mask any and all parsing errors!  The processing loop should catch any unhandled exception, log it, skip the message (not import it empty), and continue.